src/test/java/DNAnalyzer/CodonDataUtilsTest.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright © 2022 DNAnalyzer. Some rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * You are entirely responsible for the use of this application, including any and all activities that occur.
+ * While DNAnalyzer strives to fix all major bugs that may be either reported by a user or discovered while debugging,
+ * they will not be held liable for any loss that the user may incur as a result of using this application, under any circumstances.
+ *
+ * For further inquiries, please contact DNAnalyzer@piyushacharya.com
+ */
+
+package DNAnalyzer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+
+import DNAnalyzer.codon.CodonDataUtils;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+/**
+ * CodonDataUtilsTest contains 20 unit tests, one for each acid.
+ *
+ * @author @Speedro
+ */
+public class CodonDataUtilsTest {
```

Comment:
@Verisimilitude11 Ok, I am not sure about the CodeSceneDelta analyzis fail as the strings are valid inputs for the test case. Maybe this analyzis could be softened for test classes or something.

Reply:
Or the dna parts (ATT, ATC etc.) could be turned into String constants. Not sure it that helps though, but maybe could.

Reply:
Yeah don't worry about CodeScene, they don't have the most reliable reporting. I'll merge later tomorrow after some testing.